src/endpoint/s3/s3_rest.js

```diff
@@ -231,63 +231,65 @@ function authenticate_request(req, res) {
     }
 }
 
-function parse_op_name(req) {
-    const virtual_host_suffix = req.virtual_host_suffix;
-    const method = req.method.toLowerCase();
-    const host = req.headers.host.split(':')[0]; // cutting off port
-    const url = req.url;
-
-    var bucket = '';
-    var key = '';
-    var i;
-
-    // see http://docs.aws.amazon.com/AmazonS3/latest/dev/VirtualHosting.html
-    const is_path_style = !host ||
-        !virtual_host_suffix || // added case for when no DNS name is defined - if we work with IP always using path-style
-        virtual_host_suffix === '.' + host ||
-        net.isIP(host) ||
-        host === 'localhost'; // we added this case on top of the S3 doc cases to handle requests with IP host
-
-    const is_bucket_subdomain =
-        virtual_host_suffix &&
-        host.endsWith(virtual_host_suffix) &&
-        host !== virtual_host_suffix; // non empty bucket name
-
-    if (is_path_style) {
+// We removed support for default website hosting (host value is a bucket name)
+// after discussion with eran and gay because it intreeduced a conflict.
```

Comment:
```suggestion
// after a discussion with [Ohad, Nimrod, Eran, Guy] because it introduced a conflict
// that will be resolved with Bucket Website Configuration
```